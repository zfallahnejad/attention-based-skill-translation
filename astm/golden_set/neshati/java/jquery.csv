573032,jquery
157882,jquery
315935,jquery
