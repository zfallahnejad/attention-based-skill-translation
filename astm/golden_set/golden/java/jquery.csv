573032,53,38,0.5824175824175825
157882,41,16,0.7192982456140351
315935,11,7,0.6111111111111112
